roles/rsyslog/defaults/main.yaml

```diff
@@ -383,11 +347,57 @@ rsyslog_conf_local_modules:
                   if ("mark" in rsyslog_capabilities)
                   else "absent" }}'
 
-# rsyslog_conf_common_defaults
+# __rsyslog_conf_network_modules
+#
+# List of ``rsyslogd`` modules that receive logs from remote systems over the
+# network. They are enabled by the ``network`` capability.
+__rsyslog_conf_network_modules:
+
+  - name: 'network-modules'
+    type: 'modules'
+    state: '{{ "present"
+              if ("network" in rsyslog_capabilities)
+              else "absent" }}'
+    sections:
+
+      - comment: 'Enable UDP support'
+        options: |-
+          module(load="imudp")
+        state: '{{ "present"
+                  if (rsyslog_send_over_tls_only)
+                  else "absent" }}'
+
+      - comment: 'Enable TCP support'
+        options: |-
+          module(load="imptcp")
+        state: '{{ "present"
+                  if (rsyslog_send_over_tls_only)
+                  else "absent" }}'
+
+      - comment: 'Enable Secure TCP support based on rsyslog_default_netstream_driver'
+        options: |-
+          module(
+            load="imtcp"
+            streamDriver.name="{{ rsyslog_default_netstream_driver }}"
+            streamDriver.mode="1"
+            streamDriver.authMode="{{ rsyslog_default_driver_authmode }}"
+          {% if rsyslog_default_driver_authmode != "anon" %}
+            {% if rsyslog_permitted_peers is string %}
+              permittedPeer="{{ rsyslog_permitted_peers }}"
+            {% else %}
+              permittedPeer=["{{ rsyslog_permitted_peers | join('","') }}"]
+            {% endif %}
+          {% endif %}
+          )
+        state: '{{ "present"
+                  if ("tls" in rsyslog_capabilities)
+                  else "absent" }}'
+
+# __rsyslog_conf_common_defaults
```

Comment:
this is a big change, what is the relation to the rest of the PR?

Reply:
It's not...  I'm removing this section.  Sorry for the noise. :(